igor-web/src/main/groovy/com/netflix/spinnaker/igor/wercker/WerckerBuildMonitor.groovy

```diff
@@ -187,11 +182,12 @@ class WerckerBuildMonitor extends CommonPollingMonitor<PipelineDelta, PipelinePo
                     runningBuilds: currentlyBuilding
                     ))
         } catch (e) {
-            log.error("Error processing runs for [{}:{}]", kv("master", master), kv("pipeline", pipeline), e)
-            if (e.cause instanceof RetrofitError) {
```

Comment:
Was this a bug to be looking at `e.cause` instead of `e`?  Does anything inside the try block ever use a retrofit class to make a http call?  I see `werckerService.getBuilds(pipeline)` but it's before the beginning of the try block.